web/themes/custom/pineapple/sass/base/forms/_form-items.scss

```diff
@@ -0,0 +1,65 @@
+.user-register-form,
+.user-login-form {
+  background-color: $white;
+  color: $dark-gray;
+  font-weight: bold;
+  font-family: "Lato", sans-serif;
+  text-transform: capitalize;
+  padding: $unit * 2;
+  font-size: 12px;
+  width: 70%;
```

Comment:
These forms should be 100% width on mobile

Reply:
done